test/e2e/kubernetes/scripts/pwquality-validate.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -x
+
+# an invalid password should fail
+echo "tooshort1@" | sudo pwscore && exit 1
+echo "password123456@J" | sudo pwscore && exit 1
```

Comment:
I tried the first two commands on a Linux box, but the first one failed?

```
$ echo "tooshort1@" | sudo pwscore && echo 1
50
1
$ echo "password123456@J" | sudo pwscore && echo 1
Password quality check failed:
 The password fails the dictionary check - it is too simplistic/systematic
```

Reply:
that's because you don't have the same `etc/security/pwquality.conf`! Yours is probably the default, which allows passwords >= 9 characters. Part of this PR is to enforce stricter password rules, >= 14 characters with at least 1 uppercase, 1 lowercase, 1 special char, 1 digit and no recognized dictionary words.

I will add this to the PR description for reference.

Reply:
Makes total sense. Let's agree to say that I was trying to test the script with the defaults to see if it did anything meaningful. 😜